scrapers/TheScoreGroup.yml

```diff
@@ -30,20 +33,44 @@ xPathScrapers:
               - regex: ..,
                 with:
           - parseDate: January 2 2006
-      Details:
-        selector: //div[@class="row"]/div/div[@class="p-desc"]/text()
-        concat: "\n"
+      Details: &details
+        selector: //div[@class="p-desc p-3"]//text()
+        postProcess:
+          - replace:
+              - regex: Read More »
+                with:
+        concat: "\n\n"
       Tags:
-        Name: //div[@class="row"]/div/div[@class="mb-3"]/a/text()
+        Name: //div[@class="mb-3"]/a/text()
       Performers: &performersAttr
         Name: //div[@class="stat mb-2"]/span[contains(.,"Featuring:")]/../span[@class="value"]/a/text()
       Image:
-        selector: //meta[@itemprop="image"]/@content
-  # Enable this post process if you want better image quality but sometimes it can fail
-  #        postProcess:
-  #          - replace:
-  #            - regex: _lg
-  #              with: _x_800
+# This selector scrapes the canonical scene page cover image
+        selector: //script[contains(text(), "poster")]
+        postProcess:
+          - replace:  
+              - regex: ^.+poster.+'(.+jpg)'.+$
+                with: $1
+# This Selector scrapes the image posted on social media sites
+#        selector: //meta[@itemprop="image"]/@content
+# Enable this post process if you want better image quality but sometimes it can fail
+#          postProcess:
+#            - replace:
+#                - regex: _lg
+#                  with: _x_800
+      URL: &urlAttr
+        selector: $url
+        # Sanitize the URL
+        postProcess:
+          - replace:
+              - regex: (https?:\/\/(?:www\.)?[0-9A-za-z-]+\.com\/[0-9A-Za-z-_]+\/[0-9A-Za-z-_]+\/\d+)\/?.*
+                with: $1
```

Comment:
The post-processing should not be necessary here: the canonical URL by definition does not contain any tracking parameters so we can use it as it's written

Reply:
Manually obtaining a URL from their site always contains the tracking parameters in my experience. Users could sanitize it themselves but I'm betting they're not going to bother. This is why I'm sanitizing it in the scraper.

Reply:
I believe all of their anchor tags have tracking parameters, but none of the pages I've checked have had tracking parameters in the `<link rel="canonical">` tag: I only bothered to point this out because they also consider the trailing slash to be canonical and your reconstruction omits that 😅

Reply:
Oh, snap. I understand what you're communicating now. Sanitizing the URL is uncessary because I am already obtaining the clean version from `//link[@rel="canonical"]/@href`. 🙃😂 Major brainfail there.

I think the sanitization bit was included because as I updated my very old scraper I forgot to omit that vestigal section and just pieced it together with the new one. 💩